api/src/migrations/1645259040554-ImpactStoredFunctions.ts

```diff
@@ -258,8 +258,53 @@ export class ImpactStoredFunctions1645259040554 implements MigrationInterface {
     END;
   $$
   LANGUAGE plpgsql;
-
   `);
+
```

Comment:
Stored functions added on this migration won't work, and I am really curious on how it worked for you (there is a reason but it is a weird one) 

Let's talk in slack about how migrations work :)